seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -1143,7 +1143,7 @@
         }
       ],
       "description": [
-        "<code>Google Fonts</code> is a great way to find free fonts to use in a project.",
+        "<code><a href='https://fonts.google.com/' target='_blank'>Google Fonts</a></code> is a great way to find free fonts to use in a project.",
```

Comment:
I'm not sure about this formatting. I tried removing the wrapping `<code>` locally, but it broke the paragraph into separate lines.

Ideally, I think this link should behave like a `display: inline` and without being wrapped in a `<code>` tag.